src/index.js

```diff
@@ -0,0 +1,66 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './style.css';
+import addNewItem from './addTodo.js';
+import clearCompletedTodos from './clearCompletedTodos.js';
+import removeTodo from './removeTodo.js';
+import editTodo from './editTodo.js';
+import completeTask from './completeTask.js';
+import displayTodos from './displayTodos.js';
+
+const addNewTodo = document.querySelector('.addNewTodo');
+const todoList = document.querySelector('#todoList');
+const clearCompleted = document.querySelector('#clearCompleted');
+// const todos = JSON.parse(localStorage.getItem('todos'));
```

Comment:
You don't need to keep commented out code. Please delete this and any other commented out code.